typescript/src/schema/proto.ts

```diff
@@ -26,7 +26,17 @@ import {BundleConfig} from '../bundle.js';
 import {Options} from './naming.js';
 import {ServiceYaml} from '../serviceyaml.js';
 import protobuf from 'protobufjs';
-import protoJson from '../../../protos/protos.json' assert {type: 'json'};
```

Comment:
Why is this change needed? What's wrong with the regular import? The path is static anyway, isn't it?